modules/apps/frontend-theme/frontend-theme-classic/src/css/portlet/_portlet_decorators.scss

```diff
@@ -1,32 +0,0 @@
-.portlet-decorate {
```

Comment:
Do we need 3 types of portlet?

Reply:
I would say customers have been using this for years.. so yeah :-D

Reply:
I see, although the widgets in default page (fragments) now have these options by default in the sidebar